css/style.css

```diff
@@ -0,0 +1,546 @@
+    color: white !important;
+  }
+  
+  .magazine h6 {
+    font-size: 20px;
+    font-family: "Roboto Condensed",sans-serif;
+  }
+  
+  .editorial h5 a{
+    font-size: 24px;
+  }
+  
+  .us h5 a {
+    font-size: 22px;
+    font-family: "Roboto Condensed",sans-serif;
+    color: black;
+  }
+  
+  .us img {
+    min-height: 110px;
+    min-width: 110px ;
+  }
+  
+  .img-subs-2 {
+    top: 30px;
+    left: 110px;
+  }
+  
+  .subs-newsletter h3 {
+    font: bold 24px "Roboto Condensed",sans-serif;
+  }
+  
+  .subs-newsletter p {
+    font: 15px "Roboto Condensed",sans-serif;
+  }
+  
+  .subs-header {
+    font: 16px "Roboto Condensed",sans-serif;
+    color: #F72210;;
+  }
+  
+  
+  
+  @media screen and (max-width: 991.99px) {
+    .header-main-1,
+    .header-top-menu,
+    .header-bottom-menu {
+      display: none;
+    }
+  
+    .container-fluid {
+      padding: 0;
+      margin: 0;
+    }
+  
+    .fa-user {
+      display: none;
+    }
+  
+    .header-main-2 svg {
+      transform: scale(1.05);
+      padding-left: 80px;
+    }
+  
+    .footer-icon {
+      padding-top: 30px;
+    }
+  
+    section.main-content {
+      margin-top: 0;
+    }
+  
+    .opinion-img > div,
+    .opinion-img-sm .signup{
+      margin: 0 auto !important;
+      max-width: 300px;
+    }
+  
+    .feature-stories img {
+      max-width: 100%;
+      height: 230px !important;
+    }
+  
+    .top-stories img {
+      height: 275px !important;
+    }
+  }
+  
+  @media screen and (min-width: 992px) {
+    .header-main-2 {
+      display: none;
+    }
+  
+    .header-main-1-lg > div {
+      padding-top: 40px;
+    }
+  
+    .top-stories-sec img{
+      height: 300px;
+    }
+  
+    .newslet-img {
+      min-width: 70px;
+      max-width: 185px;
+      min-height: 344px;
+      max-height: 432px;
+    }
+  }
+  
+  @media screen and (max-width: 767.99px) {
+    .header-main-2 svg {
+      margin-left: -30px;
+      height: 30px;
+      width: 250px;
+      margin-top: 25px;
+      position: static;
+    }
+  
+    .header-main-3 svg {
+      margin-bottom: 10px;
+    }
+  
+    .header-main-3 .person-icon-acc {
+      margin-top: 10px !important;
+    }
+  
+    .header-main-2 img,
+    .header-main-3 img {
+      width: 40px;
+      height: 40px;
+    }
+  
+    .login {
+      display: none;
+    }
+  
+    .subscribe {
+      border-radius: 15px;
+      padding: 5px;
+      background-color: #ffe400;
+      margin-right: 5px;
+      margin-left: 10px;
+      font-size: 12px;
+    }
+  
+    .social-icons {
+      float: left !important;
+    }
+  
+    .copy-right {
+      position: relative;
+      font-size: 12px;
+      top: 3px;
+      padding-bottom: 5px;
+    }
+  
+    .fa-user {
+      display: inline;
+      margin-right: -50px;
+    }
+  
+    .feature-stories img,
+    .top-stories img,
+    .debate img,
+    .my-turn img,
+    .cult-and-tra img {
+      max-width: 100% !important;
+      height: 100% !important;
```

Comment:
- [ ] Try not to use `!important` if it can be avoided. This can be done with bootstrap classes.